go.mod

```diff
@@ -16,13 +16,12 @@ require (
 	github.com/mattn/go-colorable v0.0.9
 	github.com/mattn/go-isatty v0.0.7 // indirect
 	github.com/mattn/go-sqlite3 v1.10.0 // indirect
-	github.com/measurement-kit/go-measurement-kit v0.0.0-20190509222408-0d0cd3233336
-	github.com/oschwald/geoip2-golang v1.2.1
-	github.com/oschwald/maxminddb-golang v1.3.0 // indirect
+	github.com/measurement-kit/engine v0.0.0-20190516143312-a38282337fe6 // indirect
+	github.com/measurement-kit/go-measurement-kit v0.0.0-20190521082856-635e836bbb9d
+	github.com/ooni/probe-engine v0.0.0-20190522105151-0c60545f09ea
```

Comment:
What is the difference between `measurement-kit/engine` & `ooni/probe-engine`?

Reply:
I needed to re-run `go mod tidy`, sorry for the confusion :-)